util-images/access-tokens/cmd/main.go

```diff
@@ -103,5 +109,6 @@ func newConfig(tokenFile, rootCAFile string) (*rest.Config, error) {
 		TLSClientConfig: tlsClientConfig,
 		BearerToken:     string(token),
 		BearerTokenFile: tokenFile,
+		RateLimiter:     flowcontrol.NewFakeAlwaysRateLimiter(),
```

Comment:
Cannot you just set QPS: *qps here?

Reply:
It's more complicated than that:
  1. We are already rate limiting in `func worker` - we do not need another limiter here
  2. Setting a QPS would require to set Burst parameter. It raises a question what would be sane value for Burst.

Reply:
Setting burst to 1 should work, we need 1 initial request and then the the rate limiter's bucket should be refilled with the exact qps we use.

But I see your point and I agree. There is no point in having two rate-limiting mechanism.

Could you just add a comment here explaining that, basically what you wrote in point 1. ?

Reply:
Added, done.